source/_components/switcher_kis.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: page
+title: "Switcher"
+description: "Controlling your Switcher V2 Water Heater."
+date: 2018-04-10 17:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: switcher_boiler.png
+ha_category:
+  - Switch
+redirect_from:
+  - /components/switch.switcher_kis
+ha_release: "0.92"
+ha_iot_class: "Local Push"
+---
+
+This `Switcher` component allows you to control the [Switcher V2 Water Heater](https://www.switcher.co.il/).
+
+To enable it, add an entry to your `configuration.yaml` according to the following configuration instructions.
+
+To retrieve your device's details, please follow the instructions [here](https://github.com/NightRang3r/Switcher-V2-Python).
+
+<p class='note warning'>
+  Please note, the Switcher-V2-Python script is written in python 2.7 syntax, it won't run with python 3.x.
+</p>
+<p class='note warning'>
+  Please note, for the Switcher-V2-Python script to run successfully, you need to configure your device to work locally.
+</p>
+
+```yaml
+# Example of a minimal configuration.yaml entry
+switcher_kis:
+  phone_id: 'REPLACE_WITH_PHONE_ID'
+  device_id: 'REPLACE_WITH_DEVICE_ID'
+  device_password: 'REPLACE_WITH_DEVICE_PASSWORD'
+
+# Example of a full configuration.yaml entry
+switcher_kis:
+  phone_id: 'REPLACE_WITH_PHONE_ID'
+  device_id: 'REPLACE_WITH_DEVICE_ID'
+  device_password: 'REPLACE_WITH_DEVICE_PASSWORD'
+  name: 'boiler'
+  icon: 'mdi:some-icon'
+```
+
+{% configuration %}
+phone_id:
+  description: The device's phone id.
+  required: true
+  type: string
+device_id:
+  description: The device's id.
+  required: true
+  type: string
+device_password:
+  description: The device's password.
+  required: true
+  type: string
+name:
```

Comment:
Docs need to be updated to match the PR

Reply:
Thank you!
Fixed in this [commit](https://github.com/home-assistant/home-assistant.io/pull/8480/commits/94a81c0bd0189258109757ea1a112f1607056b2d).